api/ViewTransition.json

```diff
@@ -14,8 +14,8 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": false,
-            "impl_url": "https://bugzil.la/1823896"
+            "version_added": "preview",
+            "impl_url": "https://bugzil.la/1950759"
```

Comment:
Note that we generally don't keep links to resolved bugs.

For "preview" support, we could point to a bug about enabling the feature by default, but that bug does not seem to exist yet.

I'll push a fix.

Reply:
Resolved in 2c6a48025.